internal/pkg/agent/install/.flavors

```diff
@@ -0,0 +1,13 @@
+basic:
```

Comment:
What value are we getting out of making this a file and having to deal with opening+parsing it vs just embedded this list into the agent binary?

Reply:
upgrade scenario. 
if you want to change content of flavor in between upgrades you need somehow know what should be present in a new version. upgrade code is executed from stale version not the new one so embedded content would reflect state of things of version-1. if you add a new component or change layout of existing one in a new version it would be stripped and break

Reply:
it should be ok to merge if you're ok with mixing concerns. the only difference is that manifest is in versioned home after installation, but it should not matter in this case.

Reply:
Keeping it in versioned home is a benefit IMO so that it is a property of an individual agent version. I'd rather not introduce something new for this if we can handle this with an existing mechanism.

CC @pchila for opinions on use of the manifest file for defining the flavors per agent version since he implemented it originally.

Reply:
updated to use manifest in a meanwhile

Reply:
I am not against using the manifest... In the original RFC where the manifest was introduced, there was already a proposal to have  a list of inclusion/exclusion of file paths during install/upgrade (never implemented as the RFC changed based on feedback) so package manifest was already supposed to be used for this purpose.